### ISSUE TITLE
Typo in toString javadoc of @Modifiable class

### ISSUE THREAD

**Issue by jsotuyod** [immutables/immutables#320]
the javadoc says

```java
  /**
   * Generates a string represetion of this {@code User}.
   * If uninitialized, some attribute values may appear as question marks.
   * @return A string representation
   */
```

Note it says **represetion** and not **representation**

**Comment by elucash** [comment]
Corrected, thank you, will be in the next release (2.0-beta2)

**Comment by elucash** [comment]
released in 2.2 Thank you!
